src/services/api/src/queue/handle.py

```diff
@@ -7,10 +7,8 @@
 import os
 
 from ray.serve._private.client import *
```

Comment:
I'd prefer if this imported the specific methods/classes, just from a readability standpoint.